modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/verify/system/event/SystemEventVerifyProcess.java

```diff
@@ -1,85 +0,0 @@
-/**
- * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
```

Comment:
Hi @dantewang ,
I am not going to merge this change.

The reason is this will actually change the runtime environment by avoiding an registration of an OSGi service, which may affect runtime logic.

For this round, let's just remove the safe ones, meaning removing won't cause any side effect.

Tina.